cli/src/cpp/usages.hpp

```diff
@@ -199,6 +199,28 @@ static const char DURABILITY_QOS_USAGE[] =
       persistent
 )";
 
+static const char ENTITY_DEFAULT_PROFILE_USAGE[] =
+        R"(Profiles entity default profile usage:
```

Comment:
```suggestion
        R"(DDS entity default profile usage:
```

Apply this suggestion to the usage. Every `profiles entity` change for `DDS entity` (or just `entity`)